### PR TITLE
Display and add to patient med history

### PR DESCRIPTION
**Pull request by captDaylight**
For:
- https://www.notion.so/photons/Add-Medical-History-panel-to-prescribe-view-e0a3dbb669734c5ba794055943825df7?pvs=4
- https://www.notion.so/photons/Add-ability-to-add-externally-reported-medical-history-for-a-patient-522000f490ba4a01b2512da4c7407262?pvs=4

✨ _video demo_
https://www.loom.com/share/b2f3c546b60b432798bfb3e6c22c8a5d

### CODE REVIEW

**Review by jknapp25 — APPROVED** [review]
suuuuper cool, love the org setting too, great work 👏🏼

File: packages/components/src/systems/PatientMedHistory/index.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      query: GET_PATIENT_MED_HISTORY,
+      variables: { id: props.patientId }
+    });
+    if (data?.patient?.medicationHistory) {
+      setMedHistory(data.patient.medicationHistory);
+    }
+  };
+
+  const addMedHistory = async (medicationId: string) => {
+    await (client as PhotonClient)!.apollo.mutate({
```
Comment (marisbest2) [inline packages/components/src/systems/PatientMedHistory/index.tsx:80]: Why does this need the type assertion?

File: packages/components/src/utils/formatDate.ts [hunk 2]
```diff
@@ -1,22 +1,13 @@
...
+  const year = date.getFullYear();
 
-  // Check if month and day are within range
-  if (month < 1 || month > 12 || day < 1 || day > 31) {
-    return '';
-  }
-
-  // use template literals to format the date string
-  return `${year}-${month.toString().padStart(2, '0')}-${day.toString().padStart(2, '0')}`;
+  const formattedDate = `${month}/${day}/${year}`;
```
Comment (marisbest2) [inline packages/components/src/utils/formatDate.ts:11]: Why'd we change our default here?

**Review by marisbest2 — APPROVED** [review]
Two qs. but LGTM

File: packages/components/src/utils/formatDate.ts (hunk 2 above)
Comment (captDaylight) [reply packages/components/src/utils/formatDate.ts:11]: this wasn't being used anywhere inside of the `components` so I reimplemented it.

File: packages/components/src/systems/PatientMedHistory/index.tsx (hunk 1 above)
Comment (captDaylight) [reply packages/components/src/systems/PatientMedHistory/index.tsx]: this one is a true murder mystery, IDE was acting up before, but now I just removed and it's fine :|

**Pull request merged**